chat/src/main/java/com/github/twitch4j/chat/events/channel/IRCMessageEvent.java

```diff
@@ -73,6 +79,18 @@
 	 */
 	private Optional<String> payload = Optional.empty();
 
+    /**
+     * Cache: ChannelId to ChannelName
+     */
+    @ToString.Exclude
```

Comment:
Good thinking. Should we also apply `EqualsAndHashCode.Exclude`? (same goes for the second map below)

Reply:
I removed those properties from the class